processing/src/main/java/org/apache/druid/math/expr/vector/ExprEvalDoubleVector.java

```diff
@@ -69,4 +70,13 @@ public Object[] getObjectVector()
     }
     return objects;
   }
+
+  @Override
+  public boolean elementAsBoolean(int index)
+  {
+    if (nulls != null && nulls[index]) {
+      return Evals.asBoolean(0.0);
```

Comment:
This is just `return false` right?

Reply:
it is, and i suppose it is unlikely we will ever change the behavior of the `Evals.asBoolean` calls, I was just being overly cautious to match logic in `ExprEval` exactly